### PR TITLE
feat: list pool by addresses & list tokens by addresses or symbols

### PR DESCRIPTION
**Pull request by nullmastermind**
### What
- Get all pool by address list
- Get token by symbol list or address list

### CODE REVIEW

File: token.go [hunk 1]
```diff
@@ -28,6 +28,8 @@ func (i Token) String() string {
 
 type TokenListOptions struct {
 	ListOptions
+	Addresses string `url:"addresses,omitempty"`
```
Comment (himitsuko) [inline token.go]: @bluevariant
How do you think about the way below ?
We will require the end user to pass the array of the string that contains a set of the addresses. And the step to convert that array to string will be transparent with our end user.
If we do this way, we can figure out that we need an array of the address and return the list of tokens represent by this array.

**Review by himitsuko — COMMENTED** [review]
@bluevariant I think we need to add the `type` filter which was listed at

https://github.com/depocket/depocket_sdk_backend/pull/1

**Comment by nullmastermind** [comment]
@himitsuko [commit](https://github.com/depocket/go-client/pull/1/commits/098f48bb4a6c47906047f6cb77ae9b631b88f1b9) supported three ways:

1. `[]string` (E.g: `[]string{"0x00", "0x01"}`)
2. `[]common.Address` (go-ethereum) (E.g: `[]common.Address{common.HexToAddress("0x00"),common.HexToAddress("0x01")}`)
3. `string` (E.g: `"0x00,0x01"`)

How do u think?

**Comment by himitsuko** [comment]
> 1. []string (E.g: []string{"0x00", "0x01"})

I think this way is the simplest. The users don't need to transform anything. And it is easy to use and clear.

> 2. []common.Address (go-ethereum) (E.g: []common.Address{common.HexToAddress("0x00"),common.HexToAddress("0x01")})

This way have pros also cons.
- pros:
  - common.HexToAddress will help us to validate the address before our user post it to the backend.
- cons:
  - It is a bit redundant because it's enough to validate on the backend
 side
  - we need to transfrom too many things when the number of the address is increase
> 3. string (E.g: "0x00,0x01")

Go through https://github.com/depocket/go-client/pull/1#discussion_r889906974

---
@bluevariant 
I think we don't need too many options. It will make the code more confusing to understand in the long term when each other has a different implement.
I will choose 1 on my side.

**Review by himitsuko — APPROVED** [review]

**Review by haihv — APPROVED** [review]

**Pull request merged**